File: commando/test/commando/MultiplyCommandTest.java
package commando;

import org.junit.Test;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.equalTo;

public class MultiplyCommandTest {

    private final MultiplyCommand command = new MultiplyCommand();

    @Test
    public void willCalculateProduct() throws Exception {
        assertThat(command.execute("3,4"), equalTo("12"));
    }

    @Test (expected = InvalidCommandException.class)
    public void willBlowUpIfIllegalFormat() throws Exception {
        command.execute("a,4");
    }
}

File: PHP/test/Commando/MultiplyCommandTest.php
<?php
namespace Tests;

use PHPUnit\Framework\TestCase;
use Commando\MultiplyCommand;
use Commando\InvalidCommandException;

class MultiplyCommandTest extends TestCase {

    private $command;

    protected function setUp(): void {
        $this->command = new MultiplyCommand();
    }

    /** @test */
    public function willCalculateProduct() {
        $this->assertEquals("12", $this->command->execute("3,4"));
    }

    /** @test */
    public function willBlowUpIfIllegalFormat() {
        $this->expectException(InvalidCommandException::class);
        $this->command->execute("a,4");
    }
}
